sound/soc/intel/Kconfig

```diff
@@ -242,6 +249,15 @@ config SND_SOC_ACPI_INTEL_MATCH
 	# this option controls the compilation of ACPI matching tables and
 	# helpers and is not meant to be selected by the user.
 
+config SND_SOC_INTEL_NOT_RECOMMENDED
```

Comment:
There's a standard BROKEN Kconfig option. It's certainly stronger than not-recommended, but if we *really really* don't want people to use those, we can use that instead.

Reply:
@lyakh I think based on recent discussions on alsa-devel, that is still too strong at this point, so let's go with the NOT_RECOMMENDED.

Reply:
The previous version used 'UNSUPPORTED'  and some of our esteemed colleagues reacted with an anaphylactic shock ... BROKEN would not go well...

Reply:
EXPERIMENTAL is also used to describe YMMV experiences.

Reply:
EXPERIMENTAL was removed in 2012 :-)
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=5a958db311c416345a61b7bddc62a5c6c7112c7f

Reply:
Hehe, shows when I last checked :)

Reply:
I still remember discussions on transitions to 2.6 :-)